server/src/main/java/org/opensearch/ratelimiting/tracker/PerformanceTrackerSettings.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.ratelimiting.tracker;
+
+import org.opensearch.common.settings.Setting;
+import org.opensearch.common.settings.Settings;
+import org.opensearch.common.unit.TimeValue;
+
+/**
+ * Settings related to node performance trackers such as polling interval, window duration etc
+ */
+public class PerformanceTrackerSettings {
```

Comment:
We need to fix naming, these are Resource trackers, that can be used outside of rate limting. So lets call the package `resource` tracker so that it could be used broadly